fendermint/vm/interpreter/src/fvm/checkpoint.rs

```diff
@@ -116,6 +121,14 @@ where
         power_diff(curr_power_table, next_power_table)
     };
 
+    emit!(
+        VMEvent::NewBottomUpCheckpoint,
```

Comment:
What level is this emitted at exactly? And what's the name of this field?

Reply:
I used `info!` and field name is `event`.